public/javascripts/SVLabel/src/SVLabel/Main.js

```diff
@@ -174,7 +174,10 @@ function Main (params) {
         );
 
         // Speed limit
-        svl.speedLimit = new SpeedLimit(svl.panorama, svl.map.getPosition, svl.isOnboarding);
+        if (currLabel.labelType == 'NoCurbRamp') {
```

Comment:
Notice that this is some code that is meant for a different PR. You should create a different branch for each PR instead of having just one branch that you use. That way you won't have the code mixed together. This is a reminder to read over the [process for contributing code](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Process-for-contributing-new-code) page again, which should have this info!

You'll need to go through and remove the code that isn't meant for this PR before I can really review this!